packages/patternfly-4/content/design-guidelines/content/grammarandterminology.md

```diff
@@ -115,12 +129,13 @@ For consistent timestamp formatting, we recommend using the following examples:
 | Timeframe  | Usage |
 | ------------- | ----- |
```

Comment:
Does this table apply specifically for relative timestamps only? I think it does. Maybe the text above should say something like, "When reporting relative time, follow these examples:"